### ISSUE TITLE
.gitignore is missing

### ISSUE THREAD

**Issue by Astach** [Zovopwoot/ClickColor#1]
a `.gitignore` file should be created to avoid uploading uneeded file, also it can be used to ignore files containing credentials such as api key, passwords etc.
https://github.com/github/gitignore/blob/master/C.gitignore

**Comment by Zovopwoot** [comment]
If I understand, it useless to uploaded all the files project (such as librairies, dll,cmd, etc). but I need to learn 2 things : thee first is witch files are absolutely needed to be uploaded, and the second how do I create a gitignore file (on github ? or on the local git terminal ?)

**Comment by Astach** [comment]
Yes, your program generates some files at compile time that you don't need to upload. Because if I want to run your project, I just execute your code and the files will be re-generated locally on my computer.

1. The .gitignore provided in the link above takes care of recognizing what needs to be uploaded or not. 
2. Github is just a `VCS`, you do not create anything here. You have to create a file called `.gitignore` at your project root.
3. Once the file is created there's something to keep in mind: Useless files are already uploaded, so how to remove them all? Because you can't guess it here you go https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore.
